extensions/labgraph_viz/setup.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+# Copyright 2004-present Facebook. All Rights Reserved.
+
+from setuptools import find_packages, setup
+
+
+setup(
+    name="labgraph_viz",
+    version="1.0.0",
+    description="Some useful visualizations for labgraph",
+    packages=find_packages(),
+    python_requires=">=3.6, <3.7",
+    install_requires=[
+        "dataclasses",
+        "labgraph",
+        "matplotlib",
+        "numpy",
+        "PyQt5-sip",
+        "pyqtgraph",
```

Comment:
I would suggest to just keep the following for now, so that the versions between LabGraph and LabGraph-Viz are compatible:
        "labgraph==1.0.2",
        "PyQt5-sip",
        "pyqtgraph",
In addition, can you also add the version you recommended to be used? (The latest version for LabGraph is added above).